eli5/keras.py

```diff
@@ -0,0 +1,293 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import numpy as np
+import keras
+import keras.backend as K
+from keras.models import Model
+from keras.layers import Layer, Lambda
+from keras.preprocessing.image import load_img, img_to_array, array_to_img
+
+from eli5.base import Explanation
+from eli5.explain import explain_prediction
+
+
+DESCRIPTION_KERAS = """Grad-CAM visualization for image input; output is images"""
```

Comment:
> output is images

I'd thought that output is literally an image, and not an explanation object with image and heatmap attributes. I think it would be better to either drop the bit about the output, or rephrase it.

Reply:
https://github.com/TeamHG-Memex/eli5/pull/315/commits/2cf16366cf207ebd3d625ef9ce0aa97a40983762